### PR TITLE
Remove unnecessary functions.

### PR DESCRIPTION
**Pull request by rvye**

### CODE REVIEW

**Comment by rvye** [comment]
Removes randomInt & randomFloat function as they are not used very many times, just replaces them with what the function actually does.

File: index.js [hunk 1]
```diff
@@ -20,26 +20,18 @@ function setup() {
   canvas.height = canvas2.height = ch;
   
   for (var i = 0; i < maxColums ; i++) {
-    fallingCharArr.push(new Point(i * fontSize, randomFloat(-500, 0)));
+    fallingCharArr.push(new Point(i * fontSize, Math.random() * (-500 - 0) + 0));
   }
 }
 
-function randomInt(min, max) {
```
Comment (rvye) [inline index.js:27]: Only used once

File: index.js [hunk 2]
```diff
@@ -20,26 +20,18 @@ function setup() {
...
-
 function Point(x, y) {
   this.x = x;
   this.y = y;
 }
 
 Point.prototype.draw = function (ctx) {
-  this.value = charArr[randomInt(0, charArr.length - 1)].toUpperCase();
-  this.speed = randomFloat(1, 5);
+  this.value = charArr[Math.floor(Math.random() * (0, charArr.length - 1) + charArr.length - 1)].toUpperCase();
```
Comment (rvye) [inline index.js:33]: randomInt func

File: index.js [hunk 3]
```diff
@@ -20,26 +20,18 @@ function setup() {
...
+  this.speed = Math.random() * (1 - 5) + 5;
```
Comment (rvye) [inline index.js:34]: randomFloat func

File: index.js [hunk 4]
```diff
@@ -51,8 +43,8 @@ Point.prototype.draw = function (ctx) {
 
   this.y += this.speed;
   if (this.y > ch) {
-    this.y = randomFloat(-100, 0);
-    this.speed = randomFloat(2, 5);
+    this.y = Math.random() * (-100 - 0) + 0;
```
Comment (rvye) [inline index.js:46]: randomFloat func

File: index.js [hunk 5]
```diff
@@ -51,8 +43,8 @@ Point.prototype.draw = function (ctx) {
...
+    this.speed = Math.random() * (2 - 5) + 5;
```
Comment (rvye) [inline index.js:47]: randomFloat func

File: index.js [hunk 6]
```diff
@@ -20,26 +20,18 @@ function setup() {
   canvas.height = canvas2.height = ch;
   
   for (var i = 0; i < maxColums ; i++) {
-    fallingCharArr.push(new Point(i * fontSize, randomFloat(-500, 0)));
+    fallingCharArr.push(new Point(i * fontSize, Math.random() * (-500 - 0) + 0));
```
Comment (rvye) [inline index.js:23]: randomFloat func

File: index.js [hunk 7]
```diff
@@ -20,26 +20,18 @@ function setup() {
...
   }
 }
 
-function randomInt(min, max) {
-  return Math.floor(Math.random() * (max - min) + min);
-}
-
-function randomFloat(min, max) {
-  return Math.random() * (max - min) + min;
```
Comment (rvye) [inline index.js:32]: not used very many times, redundant.

**Comment by rvye** [comment]
also can u put ur number on ur profile readme or smth im trying to msg u

**Review by aaditkumar2009 — APPROVED** [review]
Ok!

**Comment by aaditkumar2009** [comment]
i was litreally studying lmao and i got your notif

**Comment by aaditkumar2009** [comment]
give me a min lemme just add my number on my profile

**Comment by aaditkumar2009** [comment]
done

**Comment by aaditkumar2009** [comment]
Ok give me a min lemme check if it works or not

**Comment by aaditkumar2009** [comment]
Nope it does not

**Comment by rvye** [comment]
what's the error

**Comment by rvye** [comment]
also, do you have a phone number that works in the US?

**Comment by aaditkumar2009** [comment]
Yes?

**Comment by rvye** [comment]
wait when did i send it

**Comment by rvye** [comment]
i didnt get it

**Comment by rvye** [comment]
i used ur whatsapp num

**Comment by aaditkumar2009** [comment]
IMG_0945

**Comment by rvye** [comment]
yeah i didnt geti t back hold on

**Comment by aaditkumar2009** [comment]
Is there a problem with your phone?

**Comment by rvye** [comment]
its at a repair shop

**Comment by aaditkumar2009** [comment]
Oh thats why alr

**Comment by rvye** [comment]
did u get that txt

**Comment by aaditkumar2009** [comment]
Why don't you try some other app if this doesn't work?

**Comment by aaditkumar2009** [comment]
Instead of sms

**Comment by rvye** [comment]
im using a website lmfao

**Comment by rvye** [comment]
idk what 2 use

**Comment by aaditkumar2009** [comment]
Xd

**Comment by aaditkumar2009** [comment]
Btw should I delete the pic now?

**Comment by rvye** [comment]
if yhou wanat

**Comment by rvye** [comment]
what could we use to chat

**Comment by aaditkumar2009** [comment]
Yeah, can't you access discord?

**Comment by rvye** [comment]
no i dont have my phone

**Comment by aaditkumar2009** [comment]
Damnit

**Comment by rvye** [comment]
what else is there to chat

**Comment by aaditkumar2009** [comment]
Uhh why don't we just use element?

**Comment by aaditkumar2009** [comment]
Or any alternative

**Comment by rvye** [comment]
link it

**Comment by aaditkumar2009** [comment]
One sec

**Comment by aaditkumar2009** [comment]
[](https://element.io/)

**Comment by rvye** [comment]
ifound it

**Comment by rvye** [comment]
ok how do i add u

**Comment by rvye** [comment]
@rvye:matrix.org

**Comment by aaditkumar2009** [comment]
One sec lemme download it I had deleted it

**Comment by rvye** [comment]
hurry up!

**Comment by aaditkumar2009** [comment]
Almost done!

**Comment by aaditkumar2009** [comment]
Logging in

**Comment by aaditkumar2009** [comment]
Check

**Pull request merged**